src/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -365,90 +389,89 @@ public boolean getFollowRedirects() {
+                        headers[i].getValue());
+            }
         }
         if (counter == 0) {
-    		Log_OC.d(TAG + " #" + mInstanceNumber, "No cookie at request before");
+            Log_OC.d(TAG + " #" + mInstanceNumber, "No cookie at request before");
         }
-	}
+    }
 
     private void logCookiesAtState(String string) {
         Cookie[] cookies = getState().getCookies();
         if (cookies.length == 0) {
-    		Log_OC.d(TAG + " #" + mInstanceNumber, "No cookie at STATE before");
+            Log_OC.d(TAG + " #" + mInstanceNumber, "No cookie at STATE before");
         } else {
-    		Log_OC.d(TAG + " #" + mInstanceNumber, "Cookies at STATE (before)");
-	        for (int i=0; i<cookies.length; i++) {
-	    		Log_OC.d(TAG + " #" + mInstanceNumber, "    (" + i + "):" +
-	    				"\n        name: " + cookies[i].getName() +
-	    				"\n        value: " + cookies[i].getValue() +
-	    				"\n        domain: " + cookies[i].getDomain() +
-	    				"\n        path: " + cookies[i].getPath()
-	    				);
-	        }
+            Log_OC.d(TAG + " #" + mInstanceNumber, "Cookies at STATE (before)");
+            for (int i = 0; i < cookies.length; i++) {
+                Log_OC.d(TAG + " #" + mInstanceNumber, "    (" + i + "):" +
+                    "\n        name: " + cookies[i].getName() +
+                    "\n        value: " + cookies[i].getValue() +
+                    "\n        domain: " + cookies[i].getDomain() +
+                    "\n        path: " + cookies[i].getPath()
+                );
+            }
         }
-	}
+    }
 
-	private void logSetCookiesAtResponse(Header[] headers) {
+    private void logSetCookiesAtResponse(Header[] headers) {
         int counter = 0;
-        for (int i=0; i<headers.length; i++) {
-        	if (headers[i].getName().toLowerCase().equals("set-cookie")) {
-        		Log_OC.d(TAG + " #" + mInstanceNumber, 
-        				"Set-Cookie (" + counter++ + "): " + headers[i].getValue());
-        	}
+        for (int i = 0; i < headers.length; i++) {
+            if (headers[i].getName().toLowerCase().equals("set-cookie")) {
+                Log_OC.d(TAG + " #" + mInstanceNumber,
+                    "Set-Cookie (" + counter++ + "): " + headers[i].getValue());
+            }
         }
         if (counter == 0) {
-    		Log_OC.d(TAG + " #" + mInstanceNumber, "No set-cookie");
+            Log_OC.d(TAG + " #" + mInstanceNumber, "No set-cookie");
         }
-        
-	}
-	
-	public String getCookiesString() {
-		Cookie[] cookies = getState().getCookies();
-		String cookiesString = "";
-		for (Cookie cookie : cookies) {
-			cookiesString = cookiesString + cookie.toString() + ";";
+    }
 
-			// logCookie(cookie);
-		}
+    public String getCookiesString() {
+        Cookie[] cookies = getState().getCookies();
+        String cookiesString = "";
+        for (Cookie cookie : cookies) {
+            cookiesString = cookiesString + cookie.toString() + ";";
+
+            // logCookie(cookie);
```

Comment:
To use it in development context when things go really, really bad. In general, we don't want to log them for privacy reasons.